File: backend/sts/Ref/src/main/java/com/mysite/ref/dto/IngredientResponseDTO.java
package com.mysite.ref.dto;

import java.time.LocalDate;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class IngredientResponseDTO {
    private int ingredientsId;
    private String ingredientsName;
    private int quantity;
    private LocalDate expirationDate;
    private int storageLocation;
    private String refrigeratorName; // Refrigerator 엔티티의 이름을 저장할 필드
    private String category;
    private String note;
    
}

File: app/src/main/java/com/example/refrigeratormanager/ingredients/IngredientResponseDTO.kt
package com.example.refrigeratormanager.ingredients

data class IngredientResponseDTO(
    val ingredientsId: Int,
    val refrigeratorId: Int,
    val ingredientsName: String,
    val quantity: Int,
    val expirationDate: String,
    val storageLocation: Int,
    val refrigeratorName: String,
    val category: String,
    val note: String?
)